src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,48 @@
-export const Tabs = () => {};
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (newId: string) => void,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId);
+
+  if (!selectedTab) {
+    onTabSelected('tab-1');
+  }
+
+  const hendlerOnTabSelected = (tabId: string) => {
+    if (tabId !== selectedTabId) {
+      onTabSelected(tabId);
+    }
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li // eslint-disable-line
```

Comment:
Avoid using 'eslint-disable-line' if possible. In this case, you can get rid of the comment by wrapping the li element with React.Fragment and using the key prop on the React.Fragment.